CanHazFunny/CanHazFunny/IJokePrint.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+public interface IJokePrint
+{
+    public abstract void PrintJoke(String joke);
```

Comment:
Should be the primative string, instead of the class String
```suggestion
    public abstract void PrintJoke(string joke);
```

Reply:
done